src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/FeedbackReviews/FeedbackReviewer/FeedbackReviewer.tsx

```diff
@@ -0,0 +1,137 @@
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Button } from '@mui/material';
+import { format } from 'date-fns';
+
+import { getDictionaryText } from 'shared/utils';
+import { Svg } from 'shared/components/Svg';
+import {
+  StyledBodyMedium,
+  StyledFlexTopStart,
+  StyledTitleBoldMedium,
+  theme,
+  variables,
+} from 'shared/styles';
+import { CollapsedMdText } from 'modules/Dashboard/features/RespondentData/CollapsedMdText';
+import { DateFormats } from 'shared/consts';
+
+import {
+  StyledItem,
+  StyledRemoveWrapper,
+  StyledReviewer,
+  StyledShowMoreWrapper,
+  StyledToggleButton,
+} from './FeedbackReviewer.styles';
+import { FeedbackReviewerProps } from './FeedbackReviewer.types';
+import { getResponseItem } from './FeedbackReviewer.utils';
+import { MIN_ANSWERS_COUNT_TO_SHOW } from './FeedbackReviewer.const';
+import { RemoveReviewPopup } from '../RemoveReviewPopup';
+
+export const FeedbackReviewer = ({
+  review,
+  reviewer,
+  isCurrentUserReviewer,
+  reviewId,
+  createdAt,
+  onReviewerAnswersRemove,
+  error,
+  isLoading,
+  'data-testid': dataTestid,
+}: FeedbackReviewerProps) => {
+  const { t } = useTranslation('app');
+  const [isOpen, setIsOpen] = useState(false);
+  const [showAllAnswers, setShowAllAnswers] = useState(false);
+  const [numAnswersToShow, setNumAnswersToShow] = useState(MIN_ANSWERS_COUNT_TO_SHOW);
+  const [removePopupVisible, setRemovePopupVisible] = useState(false);
+
+  const toggleIsOpen = () => setIsOpen((prevState) => !prevState);
```

Comment:
Could you please introduce the helper 
```
export const toggleState = (setState: Dispatch<SetStateAction<boolean>>) => () =>setState((prevState) => !prevState);
```
 and update in all places where it used (I see it in 15 places globally)
For ex.:
`const toggleIsOpen = () => setIsOpen((prevState)=> !prevState);`
 => `const toggleIsOpen = toggleState(setIsOpen)`
or 
`setIsOpen((prevState) => !prevState);` 
=> `toggleState(setIsOpen)()`